heron/schedulers/src/java/org/apache/heron/scheduler/kubernetes/V1Controller.java

```diff
@@ -297,15 +296,26 @@ void deleteStatefulSet() {
             + "] in namespace [" + getNamespace() + "] is deleted.");
   }
 
-  protected List<String> getExecutorCommand(String containerId) {
+  protected List<String> getExecutorCommand(String containerId, int numOfInstances) {
+    final Config configuration = getConfiguration();
+    final Config runtimeConfiguration = getRuntimeConfiguration();
     final Map<ExecutorPort, String> ports =
         KubernetesConstants.EXECUTOR_PORTS.entrySet()
             .stream()
             .collect(Collectors.toMap(Map.Entry::getKey,
                 e -> e.getValue().toString()));
 
-    final Config configuration = getConfiguration();
-    final Config runtimeConfiguration = getRuntimeConfiguration();
+    if (TopologyUtils.getTopologyRemoteDebuggingEnabled(Runtime.topology(runtimeConfiguration))
+            && numOfInstances != 0) {
```

Comment:
Do we want to get rid of this magic number here in the code? It's readable to me, but may not be to others.

Reply:
I copied this code from another scheduler. If we change it, we should fix it everywhere. Any name suggestions? My gut feeling is that this one might be obvious, so ok to keep as is.